flow-typed/redux.js

```diff
@@ -0,0 +1,54 @@
+declare module 'redux' {
```

Comment:
wow this looks amazing. I don't see it in the `react-redux` repository--what's the recommended way to annotate the `export default connect(...` line until then? 

I've found it real funny that there's no idiomatic way to bet the benefit of a typed connected component where you use the resulting react component. There's very little information around about it--this is in fact the only information. What I've started to do is:

```
type HamburgerType = ({ name: string }) => React$Element<*>
export default (connect(null, { toggleSidebar })(Hamburger): HamburgerType)
```

i.e. a typecast but that doesn't seem right. Using the `Connector` type seems right. Does that properly trigger flow validation on usage of the resulting react component? I.e. if it's missing props (`ownProps`) or if they are incorrect?